### PR TITLE
Fix #456 - Broken Links in Docs

### LINKED ISSUE #456: Broken Links in the Documentation
**Broken Links in the Docs**

*Fix broken links*
roshinis78@22bf2c31e8b5896c7bc4e0e4b063cb5699cbd6a1
- Fixed broken relative link to `permission_relationships.yaml` in the Permissions Mapping Usage docs.
- Replaced broken links referencing Okta source code with links to the Okta API docs.

*Update additional links to be more informative*
roshinis78@d8167d0502407f0bd0bb0c2aa4381727a8d2a106
- Updated Okta schema links to point to the corresponding objects in the Okta API docs.

*Version Info*
Cartography commit hash: 19a06766e304d657d956246179a2bb01a6d9aef6

### PR DESCRIPTION
**Pull request by roshinis78**
Fixes #456

### CODE REVIEW

File: docs/schema/okta.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 ## OktaOrganization
 
-Representation of a Okta [Organization](https://developer.okta.com/docs/concepts/okta-organizations/).
+Representation of a [Okta Organization](https://developer.okta.com/docs/concepts/okta-organizations/).
```
Comment (achantavy) [inline docs/schema/okta.md:28]: While we're updating the docs, we might as well make the grammar correct :)

Can you do us a huge favor and change 'a' to 'an'? :)

**Review by achantavy — COMMENTED** [review]
I'm being nitpicky.

File: docs/schema/okta.md (hunk 1 above)
Comment (roshinis78) [reply docs/schema/okta.md:28]: Haha yes I can definitely do that :)

**Review by achantavy — APPROVED** [review]
Thanks @roshinis78!

**Pull request merged**